definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -118,30 +118,51 @@ declare class moment$MomentDuration {
 }
 declare class moment$Moment {
   static ISO_8601: string;
+  static (): moment$Moment;
   static (
-    initDate: ?Object | number | Date | Array<number> | moment$Moment | string
+    initDate: moment$MomentOptions | number | Date | Array<number> | moment$Moment | string
   ): moment$Moment;
+  static (string: string): moment$Moment;
```

Comment:
Would the line above not encompass this one? Seeing as `string` is an option for `initDate` as well.